packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -205,6 +208,8 @@ export async function copyProjectTemplateAndReplace(
     namespaceCpp: namespaceCpp,
     languageIsCpp: language === 'cpp',
 
+    cliVersion: await getVersionOfNpmPackage('@react-native-windows/cli'),
```

Comment:
This should be the version of `react-native-windows` as that's where the template came from. It's not the CLI version.